jekyll/_cci2/remote-docker-images-support-policy.adoc

```diff
@@ -35,19 +35,19 @@ We support various tags for the remote Docker environment to allow you to choose
 
 For the latest major version of Docker:
```

Comment:
We should confirm with support updated policy before touching this multiple times